### DISCUSSION TITLE
How can i publish my NPM package so that only i can use it?

### DISCUSSION THREAD

**Discussion by MZRCode** [community/community#102013] [Packages] [answered]
### Select Topic Area

Question

### Body

The reason why i want to publish this way is that i cannot try my Type definitions in my project, so how can i publish a version that **only i** can use?

**Comment by sbk2k1** [comment, marked as answer]
I think you can check this guide out - https://docs.npmjs.com/creating-and-publishing-private-packages (Probably needs a paid account/org)
@MZRCode
